test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -0,0 +1,271 @@
+#!groovy
+import groovy.json.JsonSlurperClassic
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.rules.RuleChain
+import org.yaml.snakeyaml.Yaml
+import util.BasePiperTest
+import util.JenkinsEnvironmentRule
+import util.JenkinsDockerExecuteRule
+import util.JenkinsLoggingRule
+import util.JenkinsShellCallRule
+import util.JenkinsStepRule
+import util.JenkinsWriteFileRule
+import util.Rules
+
+import static org.junit.Assert.assertTrue
+import static org.junit.Assert.assertEquals
+
+class CloudFoundryDeployTest extends BasePiperTest {
+
+    private ExpectedException thrown = ExpectedException.none()
+    private JenkinsLoggingRule jlr = new JenkinsLoggingRule(this)
```

Comment:
I would much appreciate if we used the long form for the identifiers here, as I find those acronyms incredibly hard to comprehend.

Reply:
this is in line with all other steps ...